src/vs/workbench/contrib/accessibility/browser/accessibilityConfiguration.ts

```diff
@@ -148,6 +152,14 @@ const configuration: IConfigurationNode = {
 			],
 			tags: ['accessibility']
 		},
+		[AccessibilityVoiceChatSettingId.ListeningDuration]: {
+			'markdownDescription': localize('voiceChat.listeningDuration', "The duration in milliseconds to listen for voice chat commands. Set to 0 to disable."),
```

Comment:
Overall this is not listening duration but auto-accept after pause duration no?

Reply:
I thought that was understood, but I do see it can be confusing I will reword the description so that it is more clear to the user